### DISCUSSION TITLE
Problem with contact form

### DISCUSSION THREAD

**Discussion by sopkomir** [automadcms/discussions#10] [General]
Hi, the contact form on [www.sopcode.com](http://www.sopcode.com/) won't let me send messages to any email I enter into it since July this year, I have the latest stable version there. In June it was still working, although it was throwing to spam in the case of GMail - that's why I have an email set up there on my own domain.

**Comment by marcantondahmen** [comment]
Hi, did you change the mail config? Or did the sendmail config on your server change?

**Reply by sopkomir** [reply]
I checked, it doesn't work :( - not even when I enter my email inbox under GMail.

**Comment by sopkomir** [comment]
I have now installed the alpha version - the email form works there with my GMail address.
